asm/project/enable-services.yaml

```diff
@@ -1,85 +0,0 @@
-# Copyright 2020 Google LLC
```

Comment:
why delete this file?

Reply:
The Deployment manager has already enabled all these APIs before installing ASM.

Reply:
cool

Reply:
done